components/ActionSheet/src/ActionSheetThemer/MDCActionSheetScheme.h

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#import "MaterialColorScheme.h"
-#import "MaterialTypographyScheme.h"
+@import MaterialComponents.MaterialColorScheme;
```

Comment:
Please make sure that this compiles and links using the internal build systems. Although we don't expect any clients to use Alpha components directly, we would want to catch build errors early in development and should expect Alpha components to build using our internal tooling.